src/typings/shims.d.ts

```diff
@@ -0,0 +1,5 @@
+import * as _d3 from 'd3';
+
+declare global {
```

Comment:
if we check this in, it needs a bunch of comments explaining what this is, how to use it, how to formulate one for a different library, etc. - since this is the canonical example

Reply:
I will add some comments